client/my-sites/marketing/do-it-for-me/difm-lite-in-progress.tsx

```diff
@@ -47,36 +47,29 @@ function DIFMLiteInProgress( { siteId }: DIFMLiteInProgressProps ): React.ReactE
 		const tracksName = hasEmailWithUs
 			? 'calypso_difm_lite_in_progress_email_manage'
 			: 'calypso_difm_lite_in_progress_email_cta';
-		recordTracksEvent( tracksName, {
-			domain: domainName,
-		} );
+		dispatch(
+			recordTracksEvent( tracksName, {
+				domain: domainName,
+			} )
+		);
 	};
 
 	return (
 		<div>
 			<EmptyContent
-				title={ translate( 'Hang on! Our experts are building your site.' ) }
+				title={ translate( 'Our experts are building your site' ) }
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.